File: model/src/main/java/cn/cola/model/dto/post/PublishPostDTO.java
package cn.cola.model.dto.post;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * 发布帖子DTO
 *
 * @author ColaBlack
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class PublishPostDTO {
    /**
     * 标题
     */
    private String title;

    /**
     * 内容
     */
    private String content;

    /**
     * 标签
     */
    private List<String> tags;

}

File: model/src/main/java/cn/cola/model/dto/post/PublishPostDTO.kt
package cn.cola.model.dto.post

/**
 * 发布帖子DTO
 *
 * @author ColaBlack
 */
data class PublishPostDTO(
    /** 标题 */
    var title: String,
    /** 内容 */
    var content: String,
    /** 标签 */
    var tags: List<String>,
)